src/utils/rest-api.js

```diff
@@ -289,7 +289,11 @@ export function fetchElementsInfos(ids) {
     }).then((response) =>
         response.ok
             ? response.json()
-            : response.text().then((text) => Promise.reject(text))
+            : response
+                  .text()
+                  .then((text) =>
+                      Promise.reject(text ? text : response.statusText)
```

Comment:
can't be factorized ?  like function handleJSonResponse( response ) { 
if (response.ok) return response.json()
return response.text().then((text) => Promise.reject(text ? text : response.statusText))
}

Reply:
Yes an US was created for that

Reply:
Done